php_eval/__init__.py

```diff
@@ -0,0 +1,37 @@
+# -*- coding: utf-8 -*-
+
+"""Evaluate simple JavaScript expressions. Use it with care every keystroke triggers an evaluation."""
```

Comment:
That should be php instead of JavaScript.

Reply:
Fixed.